linalg/petsc.hpp

```diff
@@ -190,7 +193,9 @@ class PetscParMatrix : public Operator
        Otherwise, it tries to convert the operator in PETSc's classes.
 
        In particular, if @a op is a BlockOperator, then a MATNEST Mat object is
-       created using @a tid as the type for the blocks. */
+       created using @a tid as the type for the blocks.
+       Note that is @a op is already a PetscParMatrix of the same type as
```

Comment:
Looks like typo: `Note that is ...` `->` `Note that if ...`?

Reply:
fixed, thanks